app/AuthenticatedModule/Components/VotingTimeForm.php

```diff
@@ -56,6 +56,7 @@ protected function createComponentForm() : BaseForm
             }
 
             $this->redrawControl();
```

Comment:
nemá smysl, pokud je za ním redirect

Reply:
ta kompenenta neví, jestli je zatím redirect nebo ne. Tedy bych to tady nechal

Reply:
ten redirect je hned na dalším řádku :-)

Reply:
sorry, koukal jsem ve špatné větvi, upraveno